feat: adding with_deleted example (#144)
revelrylabs/ecto_soft_delete: README.md

results = Repo.all(query)
```

### Getting Deleted Rows

> To query fro items that has been deleted, use `with_deleted` 

Reviewer: ```suggestion
To query for items that have been deleted, use `with_deleted: true` 
```
